addons/project/models/project_project.py

```diff
@@ -511,6 +511,9 @@ def create(self, vals_list):
                     vals['stage_id'] = stage.id
 
         projects = super().create(vals_list)
+        for project in projects:
+            if project.privacy_visibility == 'portal' and any(user._is_portal() for user in project.partner_id.user_ids):
```

Comment:
why not directly add the customer in the project followers? I don't see why it is needed to check if the partner is linked to a portal user.

Reply:
It does not make sense to me, the project visibility could be alter after the creation of the project and so the customer will still not have access to the project in his portal. 🤔 

Also, the portal user for that customer could be created afterwards.

So, IMO, we could directly set the customer as follower of the project.